address-space-controller/src/main/java/io/enmasse/controller/ControllerChain.java

```diff
@@ -139,7 +139,9 @@ public void onUpdate(List<AddressSpace> resources) throws Exception {
 
             for (Controller controller : chain) {
                 try {
-                    controller.reconcileAll(updatedResources);
+                    controller.reconcileAll(updatedResources.stream()
+                            .filter(a -> !Controller.isDeleted(a))
```

Comment:
I will back out this change, because it disturbs the expected cleanup of realms for auth services, and shouldn't be needed.